src/content/en/updates/2017/03/chrome-58-media-updates.md

```diff
@@ -0,0 +1,193 @@
+
+<div class="attempt-right">
+<figure>
+  <img src="/web/updates/images/2017/03/media-controls.png"
+       alt="Native media controls in Chrome 58">
+  <figcaption>
+    <b>Figure 1.</b>
+    Native media controls in Chrome 58
+  </figcaption>
+</figure>
+</div>
+
+This API aims to offer a way to control the native controls elements/buttons
+that are being shown by the browser to remove some features that do not make
+sense or are not part of the expected user experience, or only whitelist a
+limited amount of features.
+
+The current implementation for now is a blacklist mechanism on native controls
+with the ability to set them directly from HTML content using the new 
+attribute `controlsList`. Check out the [official
+sample](https://googlechrome.github.io/samples/media/controlslist.html).
+
+<div class="clearfix"></div>
+
+Usage in HTML:
+
+<pre class="prettyprint lang-html">
+&lt;video controls <b>controlsList="nofullscreen nodownload noremote foobar"</b>>&lt;/video>
+</pre>
+
+Usage in Javascript:
+
+    var video = document.querySelector('video');
+    video.controls; // true
+    video.controlsList; // "nofullscreen nodownload noremote" - "foobar" not present
+    video.controlsList.remove('noremote');
+    video.controlsList; // "nofullscreen nodownload" - "noremote" not present
+    video.getAttribute('controlsList'); // "nofullscreen nodownload"
+
+[Intent to Ship](https://groups.google.com/a/chromium.org/forum/#!topic/blink-dev/tFuQd3AcsIQ/discussion) &#124;
+[Chromestatus Tracker](https://www.chromestatus.com/features/5737006365671424) &#124;
+[Chromium Bug](https://bugs.chromium.org/p/chromium/issues/detail?id=685018)
+
+## Autoplay for Progressive Web Apps added to Home screen {: #autoplay }
+
+Previously, Chrome used to block all `autoplay` with sound on Android without
+exception. This is no longer true. From now on, sites installed using the
+[improved Add to Home screen] flow are allowed to autoplay audio and video
+served from origins included in the [manifest's scope] without restrictions.
+
+<pre class="prettyprint lang-html">
+&lt;audio <b>autoplay</b> src="file-in-manifest-scope.mp4">&lt;/audio>
+</pre>
+
+[Intent to Ship](https://groups.google.com/a/chromium.org/d/topic/blink-dev/DW7_yxL_HjE/discussion) &#124;
+[Chromestatus Tracker](https://www.chromestatus.com/features/5715456904134656) &#124;
+[Chromium Bug](https://bugs.chromium.org/p/chromium/issues/detail?id=676312)
+
+## Pause autoplaying muted video when invisible {: #offscreen }
+
+As you may already know, Chrome on Android allows `muted` videos to begin playing
+without user interaction. If the video is marked as `muted` and has the `autoplay`
+attribute, Chrome starts playing the video when it becomes visible to the
+user.
+
+And now, in order to reduce the power usage, videos using the `autoplay`
+attribute will be paused when offscreen and resumed when back in view,
+following Safari iOS behavior.
```

Comment:
I think you should mention that the video will resume when visible again. It might be obvious but I can imagine a couple of developers getting worried ;)